README.md

```diff
@@ -339,3 +339,34 @@ if err != nil {
 }
 fmt.Println(keys)
 ```
+
+**Support for Adding Custom Header.**
+
+
+1) From ServiceClient (For Every API Call)
+```
+cc := kp.ClientConfig{
+		BaseURL:    "BASE_URL",
+		APIKey:     "API_KEY",
+		InstanceID: "INSTANCE_ID",
+		Headers: http.Header{
+			"Custom-Header":  {"Custom-Value"},
+		},
+	}
+```
+
+2) From ServiceCall (Per API Call)
+
+**Define Header just before the API Call and Empty out when done.**
```

Comment:
Could you change the font here as well or you can remove this (it looks redundant)